docs/en-US/component/cascader.md

```diff
@@ -44,7 +44,36 @@ cascader/last-level
 
 ## Multiple Selection
 
-Set `props.multiple = true` to use multiple selection.
+Add `:props="props"` in tag and set data `props: { multiple = true }` to use multiple selection.
```

Comment:
One last request (I am sorry that I didn't bring it up earlier)
```suggestion
Add `:props="props"` in tag and set data `props = { multiple: true }` to use multiple selection.
```

Reply:
You can simply apply my suggestion so that you don't have to update the code locally